### ISSUE TITLE
Azure provider docs outdated

### ISSUE THREAD

**Issue by vasicvuk** [crossplane/crossplane#3261]
This document is outdated:
https://github.com/crossplane/crossplane/blob/master/docs/cloud-providers/azure/azure-provider.md?plain=1#L43

grant command now requires **--scope** argument on Azure CLI and it is not documented what scope is needed. And also **--expires never** is no longer supported

**Comment by negz** [comment]
Good catch, thanks. I think we partially addressed this in https://github.com/crossplane/crossplane/pull/3166 but missed this instance.

**Comment by vasicvuk** [comment]
In general I managed to get it working by adding subscription scope and with removing of --expires never.

**Comment by ranjith-ka** [comment]
`resource mapping not found for name: "default" namespace: "" from "minikube/crossplane/provider.yaml": no matches for kind "ProviderConfig" in version "azure.crossplane.io/v1beta1"
ensure CRDs are installed first
Error from server (BadRequest): error when creating "minikube/crossplane/provider.yaml": Secret in version "v1" cannot be handled as a Secret: v1.Secret.ObjectMeta: v1.ObjectMeta.TypeMeta: Kind: Data: decode base64: illegal base64 data at input byte 0, error found in #10 byte of ...|NT_CREDS}"},"kind":"|..., bigger context ...|redentials":"${BASE64ENCODED_AZURE_ACCOUNT_CREDS}"},"kind":"Secret","metadata":{"annotations":{"kube|...`

Getting this error as well, not sure where i am missing, i will keep looking
